src/etc/inc/interfaces.inc

```diff
@@ -6799,6 +6804,72 @@ function get_wireless_modes($interface) {
 	return($wireless_modes);
 }
 
+function get_wireless_ht_modes($interface) {
+	/* return wireless HT */
+	$wireless_hts_supported = array(0 => gettext('Auto'));
+
+	$cloned_interface = get_real_interface($interface);
+
+	if ($cloned_interface && is_interface_wireless($cloned_interface)) {
+		$chan_list = "/sbin/ifconfig -v {$cloned_interface} list chan";
```

Comment:
I know we have reasonable assurance that `$cloned_interface` is valid here but I would like to see that run through `escapeshellarg()` to be safe.